device/celestica/x86_64-cel_silverstone-r0/led_proc_init.soc

```diff
@@ -0,0 +1,9 @@
+#The Port LED of Silverstone SONIC can't work well, after the issue is fixed by SAI, The led will start. 
+#led auto on; led start
+rcload /usr/share/sonic/hwsku/pre-emphasis_PAM4_optics.soc
```

Comment:
this should be separate out from led process.  Please add this into start.sh, following how it does for led.

Reply:
Do you mean this start.sh , is that right ?
https://github.com/Azure/sonic-buildimage/blob/201811/platform/broadcom/docker-syncd-brcm/start.sh

Reply:
Added in https://github.com/Azure/sonic-buildimage/pull/5424/commits/6ac2ff7fbc8e9f0075c15c79e39fed4eb64ef305
Please review it

Reply:
removed in https://github.com/Azure/sonic-buildimage/pull/5424/commits/a01897c4687e3084122b11aba40c680aa4f406d2